packages/pangea-sdk/tests/integration/asyncio/test_intel.py

```diff
@@ -180,9 +180,8 @@ async def test_ip_geolocate_default_provider(self):
         response = await self.intel_ip.geolocate(ip="93.231.182.110", verbose=True, raw=True)
         self.assertEqual(response.status, ResponseStatus.SUCCESS)
         self.assertEqual(response.result.data.country, "Federal Republic Of Germany")
-        self.assertEqual(response.result.data.city, "unna")
+        self.assertIsNotNone(response.result.data.city)
```

Comment:
self.assertIn(response.result.data.city, ["unna", "holzwickede"])

Reply:
Yes, I did that first, but I saw that postal code changed again to a 3rd value so I decided to go for another solution here. Maybe next week it's going to change again.